suse_ha-formula/suse_ha/files/cib/constraint.xml.j2

```diff
@@ -0,0 +1,12 @@
+{%- set source = 'suse_ha/files/cib/' -%}
+{%- if ctype == 'rsc_colocation' -%}
+  {%- set typetemplate = source ~ 'colocation' ~ '.xml.j2' -%}
```

Comment:
Just wondering - why not `... source ~ 'colocation.xml.j2' -%}`?

Reply:
The idea was to make it a bit easier to add templates for other types of objects contained in `<constraints>` using other respective `ctype`s in the future.

Reply:
Ok, that's a good argument :-)